hnn_core/network.py

```diff
@@ -887,7 +887,7 @@ def _instantiate_drives(self, n_trials=1):
                     drive['name']]['events'].append(event_times)
 
     def add_tonic_bias(self, *, cell_type=None, amplitude=None,
-                       t0=None, T=None):
+                       t0=None, tstop=None):
```

Comment:
@kenloi it looks like there are changes from your old pull request in this pull request. Every time you make a new PR, you should fetch a fresh copy of master and branch from there:

```sh
$ git fetch upstream master:master
$ git checkout master
$ git branch fix_average_dipoles
$ git checkout fix_average_dipoles
```

now since you already started the work, I would recommend using `git cherry-pick` in the new branch. What you should do to fix the current situation is the following:

```sh
$ git branch -m fix_average_dipoles_old
$ git fetch upstream master:master
$ git checkout master
$ git branch fix_average_dipoles
$ git checkout fix_average_dipoles
$ git cherry-pick <commit1-hash>
$ git cherry-pick <commit2-hash>
$ git push -f origin fix_average_dipoles
```

does that make sense?

Reply:
Just read this^ @jasmainak. That makes sense. I'll try and fix my current branch. Thanks!